src/apps/vpn/inspector/inspectorhandler.cpp

```diff
@@ -831,6 +831,13 @@ static QList<InspectorCommand> s_commands{
                        AddonManager::instance()->fetch();
                        return QJsonObject();
                      }},
+
+    InspectorCommand{"set_version_override", "Override the version string", 1,
```

Comment:
We already have something similar: force_update_check

Reply:
See my other comment. That doesn't work for this case.